pkg/scheduler/framework/runtime/framework.go

```diff
@@ -588,24 +588,29 @@ func (f *frameworkImpl) QueueSortFunc() framework.LessFunc {
 }
 
 // RunPreFilterPlugins runs the set of configured PreFilter plugins. It returns
-// *Status and its code is set to non-success if any of the plugins returns
+// *Status, the set of plugin names that returned Skip status
+// and its code is set to non-success if any of the plugins returns
 // anything but Success. If a non-success status is returned, then the scheduling
 // cycle is aborted.
-func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (_ *framework.PreFilterResult, status *framework.Status) {
+func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (_ *framework.PreFilterResult, _ sets.String, status *framework.Status) {
```

Comment:
Maybe we should add a dedicated field in `CycleState` to hold the set of SkipFilterPlugins

Reply:
Ah, makes sense. I didn't come up with that idea.
Let me change...

Reply:
changed implementation as suggested. 🙏